README.md

```diff
@@ -85,7 +85,7 @@ gulp.task('templates', function(){
 ### defineModule(type, [options])
 
 #### type
-Type: `String`  
```

Comment:
Can you add the whitespace back (I think you text editor automatically removed trailing whitespace)?